coordinate_test.go

```diff
@@ -42,7 +42,11 @@ func TestCoordinate_updateNodeCoordinate(t *testing.T) {
 	agent := &Agent{
 		client:            client,
 		config:            conf,
-		logger:            log.New(LOGOUT, "", log.LstdFlags),
+		logger:            hclog.New(&hclog.LoggerOptions{
+			Name:            "consul-esm",
+			Level:           hclog.LevelFromString("INFO"),
+			IncludeLocation: true,
+		}),
```

Comment:
It looks like the original logger is setting the output writer to the variable `LOGOUT`. Would it make sense to do something similar for the hclogger e.g. `Output: LOGOUT`?